### DISCUSSION TITLE
What data gets collected/stored by PocketBase?

### DISCUSSION THREAD

**Discussion by jannismilz** [pocketbase/pocketbase#1303] [Q&A] [answered]
Since this software may get used in production it would be useful to know exactly what data gets collected and processed when someone requests an API route. 
Also which data when I sign in a new user. (Is everything what it shows everything what it saves?;))

Is there such a ressource?

**Comment by ganigeorgiev** [comment, marked as answer]
The only data in addition to the one you submit are the activity logs which can be disabled by changing "Admin UI > Settings > Application > Logs max days retention" to 0.

You can find all the information that the logs have stored in the detail preview of each "Admin UI > Logs" entry:

```
ID             - unique random log request identifier
Status         - http response code (eg. 200)
Method         - http request method (eg. GET)
URL            - the requested api url url
UserAgent      - the request user agent
Auth           - auth type ("guest", "admin", "auth_record")
Referer        - from where the request originated
RemoteIP       - IP of the client (or the proxy server if you are behind loadbalancer, reverse-proxy, etc.)
UserIP         - the "real" user IP (either from CF-Connecting-IP, X-Real-IP or X-Forwarded-For headers; **this IP cannot be trusted unless you are behind a trusted proxy**)
Meta           - usually contains detailed debug information about an error response
Created/Update - log request timestamp fields
```

**Reply by morethancoder** [reply]
## Quick question

Im building a cms app using pocketbase as go extend, 
I want to get access to the logs table and display pocketbase logs in my dashboard to have control over filtering the logs and showing them in my dashboard, however I can't seem to access the logs from within the app instance, 
like for example 
`logs, _ := app.Dao().LogsStats(dbx.NewExp("created <= datetime('now', '-1 hour')"))` 
raises the following error 
```no such table: _logs; failed query: SELECT count(id) AS "total", strftime('%Y-%m-%d %H:00:00', created) AS "date" FROM "_logs" WHERE created <= datetime('now', '-1 hour') GROUP BY "date"```
